### PR TITLE
Fix unwanted indentation

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
This was not in the original driver. This is an inaccuracy.

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
@rokusottervanger, already reviewed at the German Open. I thought I applied his review, but It was missed.

**Comment by MatthijsBurgh** [comment]
ping

**Comment by rokusottervanger** [comment]
I don't remember reviewing this on RGO. See my comment.

File: dynamixel_driver/src/dynamixel_driver/dynamixel_io.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def read(self, servo_id, address, size):
             except Exception as e:
                 data = []
 
-            return data
+        return data
```
Comment (rokusottervanger) [inline dynamixel_driver/src/dynamixel_driver/dynamixel_io.py:149]: Looks like data might be referenced before assignment. What does your IDE say?

**Review by rokusottervanger — CHANGES_REQUESTED** [review]

**Comment by rokusottervanger** [comment]
Since the only change here introduces a bug, I'm closing this PR.

**Pull request closed without merging**